### DISCUSSION TITLE
Asus bios hidden list (华硕隐藏bios选项

### DISCUSSION THREAD

**Discussion by kukumo123** [PIut02/ROG-Zephyrus-G14-GA401IV-Hackintosh#3] [Q&A] [answered]
I want ask a question about GA401 bios
In my bios I can't see csm,Above 4G and some bios setting how can I unlock it?
(我在GA401的bios設定裡找不到csm和above 4g等等的選項 我該怎麼解鎖他們

**Comment by PIut02** [comment, marked as answer]
csm不需要调整，above 4g可以使用[UMAF](https://github.com/DavidS95/Smokeless_UMAF)调整，在README中有写

**Reply by kukumo123** [reply]
了解 感謝你的協助 不過我嘗試找了其他的bios選項 絕大部分沒有出現在裡面 這對安裝是否會造成問題？

**Comment by PIut02** [comment]
> 了解 感謝你的協助 不過我嘗試找了其他的bios選項 絕大部分沒有出現在裡面 這對安裝是否會造成問題？

在我测试仅需要关闭secure boot和fast boot即可
